modules/oe_theme_helper/src/Element/InPageNavigation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper\Element;
+
+use Drupal\Core\Render\Element\RenderElement;
+
+/**
+ * Provides a render element for an html element.
+ *
+ * @RenderElement("oe_theme_helper_in_page_navigation")
```

Comment:
We should use render element for the field group - it allows to render tree easily. Otherwise children field groups aren't rendered and we need some workarounds to make it work. field_group module use this way to render its groups.